mambaSharedFramework/HLSValueTypes.swift

```diff
@@ -189,6 +274,109 @@ public struct HLSClosedCaptions: FailableStringLiteralConvertible {
     }
 }
 
+/// Represents CHANNELS
+public struct HLSChannels: Equatable, FailableStringLiteralConvertible {
+    /// A count of audio channels, indicating the maximum number of independent, simultaneous audio channels present in
+    /// any Media Segment in the Rendition.
+    ///
+    /// For example, an AC-3 5.1 Rendition would have a CHANNELS="6" attribute.
+    public let count: Int
+    /// Identifies the presence of spatial audio of some kind, for example, object-based audio, in the Rendition. The
+    /// Audio Coding Identifiers are codec-specific.
+    public let spatialAudioCodingIdentifiers: [String]
```

Comment:
so these identifiers are not necessarily strongly-typed to some thing?

Reply:
It doesn't seem like it. The most popular one is `JOC`; however, the spec mentions:
> The Audio Coding Identifiers are codec-specific.

This implies that the definitions of the strings sit outside of the HLS specification and therefore could be any string... We just know that:
> An Identifier is a string containing characters from the set [A..Z], [0..9], and '-'.